service/controller/resource/configmap/desired.go

```diff
@@ -45,13 +48,37 @@ func (r *Resource) newConfigMaps(customResource v1alpha1.KVMConfig) ([]*apiv1.Co
 		return nil, microerror.Mask(err)
 	}
 
+	var release *releasev1alpha1.Release
+	{
+		releaseVersion := customResource.Labels[label.ReleaseVersion]
```

Comment:
Nit: I would like this line and the next to be in some `key.go` but can be done in a follow up :)